config/Dockerfiles/rpmbuild/rpmbuild-test.sh

```diff
@@ -1,74 +1,117 @@
 #!/bin/bash
 set +e
 
-OUTPUTDIR=/home/${fed_repo}/output
-which fedpkg
-if [ "$?" != 0 ]; then echo "ERROR: FEDPKG RPM NOT INSTALLED\nSTATUS: $?"; exit 1; fi
-# Put all output files into logs/ for rsync
-rm -rf ${OUTPUTDIR}/logs
-mkdir ${OUTPUTDIR}/logs
+RPMDIR=/home/${fed_repo}_repo
```

Comment:
Should we include some sort of validation that we have all the input env vars?

`if [ - z "${fed_repo}"] ; then
  echo "no fed_repo env var"
  exit 1
fi
`